firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Bytes.txt

```diff
@@ -0,0 +1,14 @@
+SCENARIO("Get the byte from expected position", "[bytes]")
```

Comment:
Since this file tests the `get_byte`, `set_byte<byte_index, T>`, and `set_byte<T>` functions, please use structure the SCENARIOs/GIVENs/WHENs/THENs with the standard pattern we are using for functions ([documented on Notion](https://www.notion.so/pufferfish/Testing-13ce58d72a154b6883fbd57b71d37f76#51087c7b9af449d79163521b431a2e28)).

Reply:
We should test these functions with various data types (I would suggest `uint8_t`, `uint16_t`, and `uint32_t`) and various byte indexes (I would suggest using a for loop covering all possible byte indices for each data type, as well as indices which are out-of-range, e.g. a byte index of 5 for `uint32_t`).

Reply:
@raavilagoo It looks to me like the `set_byte` tests still need `WHEN`s where the byte indices are out-of-range for `uint16_t` and `uint32_t` outputs, not just for `uint8_t` outputs.